ethereum/contracts/ETHVault.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+// Holds ETH on behalf of ETHApp
+contract ETHVault is Ownable {
+    // Accepts ETH from the caller.
+    function lock() 
+        public
+        payable
+        onlyOwner
+    {
+        require(msg.value > 0, "Value of transaction must be positive");
+    }
+
+    // Returns ETH to the caller.
+    function unlock(uint128 _amount)
```

Comment:
I think at this scope, at the level of vaults, we want to use the verbs deposit/withdraw instead of lock/unlock. Kind like of a real bank.